### PR TITLE
Migrate the tutorials from panos.pan.dev, add new redudant internet d…

### PR DESCRIPTION
**Pull request by csestito**
Migrate the tutorials from panos.pan.dev, add new redudant internet doc, update links between the docs

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: products/panos/docs/intro.md
Comment (sserrata) [inline products/panos/docs/intro.md:35]: Do we want to un-triple-tick this line too, for future?

**Review by sserrata — APPROVED** [review]
👍🏽

File: products/panos/docs/intro.md
Comment (jamesholland-uk) [reply products/panos/docs/intro.md:35]: That's me, sorry. Learned a bad pattern and replicated it everywhere until I realised it wasn't the way to do it. I mean, it works, but the code sucks. Happy to fix them all, maybe I'll wait until the other changes are merged in though? Like the markdown partials PR, my mid-Nov PR, etc? Then I'll blast away all the triple-ticks? Let me know what you think...

**Pull request merged**
